dataverse-webapp/src/main/webapp/roles-edit.xhtml

```diff
@@ -49,6 +62,7 @@
                         </div>
                     </div>
                 </div>
+                <p:watermark value="#{bundle['dataverse.permissions.roles.watermark.required']}" for="roleName roleAlias" id="requiredWatermark" />
```

Comment:
Is this applies for two inputs? If yes, then I would still add two separate tags for that.
It is notable that it is the only place (at least that I'm aware of) that we indicate that field is required by watermark on input

Reply:
I've proposed watermark to Wojtek as to better indicate that the filed is required - we may discuss this approach on daily/weekly meeting.